RULEngine/controllers/__init__.py

```diff
@@ -0,0 +1,49 @@
+
+from RULEngine.controllers.position_controllers import GrSimPositionController, RealPositionController
+from RULEngine.controllers.velocity_controllers import GrSimVelocityController, RealVelocityController
+from config.config_service import ConfigService
+
+
+class Controller:
```

Comment:
oups should rename that to avoid conflict with the controller class in the engine.